examples/common/chip-app-server/RendezvousServer.cpp

```diff
@@ -0,0 +1,101 @@
+#include "RendezvousServer.h"
```

Comment:
why isn't this code finding its way into a CHIP library?

Reply:
Currently wifi and Thread are sending their network credentials in different format. Wifi is `$ssid:$psk` in text format while Thread the network dataset in binary format.

I believe neither will be the final design for network provisioning and the D&S guys are working on the final network provision profile so we may keep those code here until that general network provision profile is implemented.

Reply:
I would suggest to add a GH issue and a todo here to remind us to merge the formats.